docs/security/fido-authentication-plugin.md

```diff
@@ -0,0 +1,111 @@
+
+After the device is registered, the server updates the `mysql.user` system table for that account with the device registration status and stores the public key and credential ID.
+
+The user must use the same FIDO device during registration and authentication. If the device is reset or the user attempts to use a different device, the authentication fails. To use a different device, the registered device must be unregistered and you must complete the registration process again.
+
+
+
+### Use FIDO authentication as the only method
+
+If FIDO is used as the only method of authentication, the method does not use a password. The authentication uses a method such as as biometric scan or security key. 
+
+The user creates an account with the `PASSWORDLESS_USER_ADMIN` privilege and the `CREATE USER` privilege. 
+
+The first element of the `authentication_policy` value must be an asterisk(*). Do not start with the plugin name. For more information, see [Configuring the Multifactor Authentication Policy](https://dev.mysql.com/doc/refman/8.0/en/multifactor-authentication.html#multifactor-authentication-policy). 
+
+You must include the `INITIAL AUTHENTICATION IDENTIFIED BY` clause in the `CREATE USER` statement. The server does accept the statement without the clause but the account is unusable because the user cannot connect to the server to register the device. 
+
+The `CREATE USER` syntax is the following:
+
+```sql
+    mysql> CREATE USER <username>@<hostname> IDENTIFIED WITH authentication_fido INITIAL AUTHENTICATION IDENTIFIED BY '<password>';
+```
+
+During registration, the user must authenticate with the password. After the device is registered, the server deletes the password and modifies the account to make FIDO the only authentication method. 
+
+## Unregister a FIDO device
+
+If the FIDO device is replaced the previous device must be unregistered and register the new device. The account owner or any user with the `CREATE USER` privilege can unregister the device. The user planning to use the device must register the new device.
+
+If the FIDO device is lost, the current device must be unregistered and a new device must be registered. The account owner or any user with the `CREATE USER` privilege can unregister the device. The user planning to use the device must register the new device.
+
+The statement to unregister a device is as follows:
+
+```sql
+    mysql> ALTER USER user {2|3} FACTOR UNREGISTER;
+```
+
+
+
+
+
+[^1]: Tech Preview Features are not yet ready for enterprise use and are not included in support via SLA. They are included in this release so that users can provide feedback prior to the full release of the feature in a future GA release (or removal of the feature if it is deemed not useful). This functionality can change from tech preview to GA.!!! note
```

Comment:
Remove the "!!! note" at the end of the paragraph.

Reply:
removed